$NetBSD: patch-ac,v 1.1 2000/08/31 14:55:32 dmcmahill Exp $

don't force the user to have latex for building the docs

--- doc/Imakefile.orig	Wed May 13 06:29:09 1998
+++ doc/Imakefile	Thu Aug 31 10:18:06 2000
@@ -32,5 +32,5 @@
  */
 
-all:: pcb.man pcb.ps pcb.html pcb.info refcard.ps
+all:: pcb.man
 
 InstallManPage(pcb,$(MANDIR))
